pkg/reconciler/channel/channel_test.go

```diff
@@ -576,6 +577,9 @@ func TestReconcile(t *testing.T) {
 	table.Test(t, MakeFactory(func(ctx context.Context, listers *Listers, cmw configmap.Watcher) controller.Reconciler {
 		ctx = channelable.WithDuck(ctx)
 		ctx = v1addr.WithDuck(ctx)
+		ctx = feature.ToContext(ctx, feature.Flags{
```

Comment:
Here you are trying to turn on OIDC for all the tests. Should we try to use the similar pattern that is used [here](https://github.com/knative/eventing/blob/5cb354f2d6408671549cf67aae7c3dac766f49b7/pkg/reconciler/broker/broker_test.go#L932-L934) in broker?

Reply:
@Leo6Leo I think currently all test cases are written assuming that the `oidc-authentication` feature is enabled.  We could refactor this if we wanted to add tests with  `oidc-authentication` feature disabled.

Reply:
@Leo6Leo Any inputs on this?

Reply:
Yes, following up on what @Leo6Leo said, we generally test many different configurations in these unit tests. In this scenario that means both with OIDC enabled and without OIDC enabled

Reply:
Updated the individual test cases in the newer commit. PTAL.